### PR TITLE
Revert file from pull request

### PR DESCRIPTION
**Pull request by at-dieulexuan**

### CODE REVIEW

File: resources/views/danabus/search/index.blade.php [hunk 1]
```diff
@@ -8,33 +8,19 @@
           <div class="col-md-6 col-md-offset-3">
```
Comment (ghost) [inline resources/views/danabus/search/index.blade.php]: Please remove one tag `</div>` in this file, it's exuberancy.

File: public/danangbus/js/dnbus.searchmodule.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+        DNBus.SearchModule.drawCircle(your_location);
+        callback({
+          lat: results[0].geometry.location.lat(),
+          lng: results[0].geometry.location.lng(),
+        });
+      }
+    });
+  },
+
+  ajaxFunction : function(data, callback){
```
Comment (ghost) [inline public/danangbus/js/dnbus.searchmodule.js]: please change `ajaxFunction` to other function name. It's very common now.

File: public/danangbus/js/dnbus.searchmodule.js [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+          marker.setMap(mymap);
+        });
+        DNBus.SearchModule.getRoutes(data, function(callbackRoutes, routes) {
+          return callback(true, routes);
+        }); 
+      },
+    });
+  },
+
+    getRoutes : function(data, callbackRoutes){
```
Comment (ghost) [inline public/danangbus/js/dnbus.searchmodule.js]: Please check format

File: public/danangbus/js/dnbus.searchmodule.js [hunk 4]
```diff
@@ -0,0 +1,130 @@
+var DNBus = {};
+DNBus.SearchModule = {
+
+  routes : [],
+
+  drawCircle : function(your_address){
```
Comment (ghost) [inline public/danangbus/js/dnbus.searchmodule.js]: Every function, you should write some comment about it.
Template comment:
```json
/**
 * This is description of function
 * @param {Number} a 
 * @param {Number} b
 * @return {Number} sum
 */
```

File: public/danangbus/js/dnbus.routesmodule.js [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+  },
+
+  /**
+  * draw Direction of routes
+  * @param {Object} directions
+  * @param {Object} directions_display
+  * @param {Array} path
+  * @param {Array} waypoints
+  */
+  drawDirection : function (directions, directions_display,path,waypoints){
```
Comment (ghost) [inline public/danangbus/js/dnbus.routesmodule.js]: Please check formate space between param.
You should take care convention for all

File: public/danangbus/js/dnbus.showlocationmodule.js [hunk 6]
```diff
@@ -0,0 +1,54 @@
+DNBus.ShowLocation = {
+
+	/**
```
Comment (ghost) [inline public/danangbus/js/dnbus.showlocationmodule.js]: Please check convention again

File: resources/views/danabus/search/index.blade.php [hunk 7]
```diff
@@ -4,52 +4,37 @@
...
-        </div>
-        <div id="mymap" style="height: 800px; width: 80%; margin: auto;">
-        </div>   
       </div>
+    </div>              
+    <div id="contact" class="contact">
+      <div class="title">
+       <h2 class="map">{{trans('index.map')}}</h2>
+      </div>
+      <div id="mymap" style="height: 800px; width: 80%; margin: auto;">
```
Comment (ghost) [inline resources/views/danabus/search/index.blade.php]: Please split css to file *.css

Comment (ghost) [reply resources/views/danabus/search/index.blade.php]: You should limit use of css inline

File: app/Http/Controllers/Danabus/SearchController.php [hunk 8]
```diff
@@ -20,20 +20,24 @@ public function index()
...
-            $id = $request->get('route_id');
-            $status = $request->get('status');
-            $condition = ['status' => $status, 'route_id' => $id];
-            $directions = Direction::where($condition)->with('stop')->get();
-            return response()->json($directions);
+            $lat = $request->get('lat');
+            $lng = $request->get('lng');
+            $raw = "(6371 * acos(cos(radians($lat)) * cos(radians(lat)) * cos(radians(lng) 
+                - radians($lng)) 
+                + sin(radians($lat))
```
Comment (at-phucnguyen) [inline app/Http/Controllers/Danabus/SearchController.php]: Phần này anh nghĩ em nên truyền binding cho $lng và $lat giống như cái `$radius` bên dưới `whereRaw`. Mục đích dùng binding là để tránh lỗi SQL Injection

**Comment by ghost** [comment]
LGTM

**Pull request merged**